apps/full-stack-tests/src/hierarchies-react/learning-snipptets/Localization.test.tsx

```diff
@@ -4,38 +4,39 @@
 *--------------------------------------------------------------------------------------------*/
 
 /* eslint-disable @typescript-eslint/naming-convention */
+/* eslint-disable no-duplicate-imports */
 import { expect } from "chai";
-import { insertPhysicalModelWithPartition } from "presentation-test-utilities";
-import { RpcConfiguration, RpcManager } from "@itwin/core-common";
-import { ECSchemaRpcImpl } from "@itwin/ecschema-rpcinterface-impl";
-import { render, waitFor } from "@testing-library/react";
-import { buildTestIModel, initialize, terminate } from "@itwin/presentation-testing";
-import { createBisInstanceLabelSelectClauseFactory, createCachingECClassHierarchyInspector } from "@itwin/presentation-shared";
-import { IModelConnection } from "@itwin/core-frontend";
+import { buildIModel } from "../../IModelUtils";
 import { SchemaContext } from "@itwin/ecschema-metadata";
-import { ECSchemaRpcInterface, ECSchemaRpcLocater } from "@itwin/ecschema-rpcinterface-common";
+import { IModelConnection } from "@itwin/core-frontend";
+import { render, waitFor } from "@testing-library/react";
+import { initialize, terminate } from "../../IntegrationTests";
+import { ECSchemaRpcLocater } from "@itwin/ecschema-rpcinterface-common";
+import { insertPhysicalModelWithPartition } from "presentation-test-utilities";
 import { createECSchemaProvider, createECSqlQueryExecutor } from "@itwin/presentation-core-interop";
 import { createLimitingECSqlQueryExecutor, createNodesQueryClauseFactory } from "@itwin/presentation-hierarchies";
-// __PUBLISH_EXTRACT_START__ Presentation.HierarchiesReact.Readme.Localization.TreeRenderer.Imports
+import { createBisInstanceLabelSelectClauseFactory, createCachingECClassHierarchyInspector } from "@itwin/presentation-shared";
+// __PUBLISH_EXTRACT_START__ Presentation.HierarchiesReact.Localization.TreeRenderer.Imports
 import { ComponentPropsWithoutRef, useCallback } from "react";
 import { Tree } from "@itwin/itwinui-react";
 // __PUBLISH_EXTRACT_END__
-// __PUBLISH_EXTRACT_START__ Presentation.HierarchiesReact.Readme.Localization.Imports
+// __PUBLISH_EXTRACT_START__ Presentation.HierarchiesReact.Localization.Tree.Imports
+import { useUnifiedSelectionTree } from "@itwin/presentation-hierarchies-react";
+// __PUBLISH_EXTRACT_END__
+// __PUBLISH_EXTRACT_START__ Presentation.HierarchiesReact.Localization.TreeRenderer.Imports
 import {
   createRenderedTreeNodeData,
   LocalizationContextProvider,
   RenderedTreeNode,
   TreeNodeRenderer,
   TreeRenderer,
-  useUnifiedSelectionTree,
 } from "@itwin/presentation-hierarchies-react";
-
 // __PUBLISH_EXTRACT_END__
 
-describe("Hierarchies-react", () => {
+describe.only("Hierarchies-react", () => {
```

Comment:
```suggestion
describe("Hierarchies React", () => {
```

Reply:
I think you were looking at outdated files as the `.only` was already removed. Changed the name though.